client/dive-common/components/Viewer.vue

```diff
@@ -77,27 +87,18 @@ export default defineComponent({
     const loadError = ref('');
     const baseMulticamDatasetId = ref(null as string | null);
     const datasetId = toRef(props, 'id');
-    const multiCamList: Ref<string[]> = ref([]);
-    const defaultCamera = ref('');
-    const currentCamera = ref('');
+    const multiCamList: Ref<string[]> = ref(['default']);
+    const defaultCamera = ref('default');
+    const selectedCamera = ref('default');
```

Comment:
I think we need to do with with something other than strings.  Using the reserved keyword "default" to name a camera causes the viewer to break.